js/makeTable.js

```diff
@@ -0,0 +1,123 @@
+function makeTable() {
```

Comment:
This should be wrapped in an AMD module if it's going to be some reusable piece of code.